website/docs/d/is_instance_groups.html.markdown

```diff
@@ -0,0 +1,87 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_is_instance_groups"
+description: |-
+  Get information about InstanceGroupCollection
+subcategory: "VPC infrastructure"
+---
+
+# ibm_is_instance_groups
+
+Provides a read-only data source for InstanceGroupCollection. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
```

Comment:
Please add the below note:
**Note:** 
VPC infrastructure services are a regional specific based endpoint, by default targets to `us-south`. Please make sure to target right region in the provider block as shown in the `provider.tf` file, if VPC service is created in region other than `us-south`.

**provider.tf**

```terraform
provider "ibm" {
  region = "eu-gb"
}
```